example/wasm/site/index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Test ml dart wasm</title>
+  <link rel="preload" href="/test.wasm" as="fetch" crossorigin>
```

Comment:
@flutter-painter it seems that it's needed to use full path here and below: `http://localhost:8080/site/test.wasm`

Reply:
Are you sure ? I think it only preloads the resource and points to where it is at in href
https://stackoverflow.com/questions/36641137/how-exactly-does-link-rel-preload-work
(this comes from the dart "official" wasm example)

Reply:
@flutter-painter I encountered an issue with running the demo pages and was only able to get them working after fixing the paths. Is it possible I missed something?

Reply:
@flutter-painter Since the root of the server is the `wasm` folder, the path is being resolved relative to `wasm`, which fails because there is no `wasm/test.wasm` path — only `wasm/site/test.wasm`. You can simply use the `/site/test.wasm` path instead: 

```
<link rel="preload" href="/site/test.wasm" as="fetch" crossorigin>
```

The same applies to the paths mentioned below.